app/komponenter/veilederhilsen/veilederhilsen.tsx

```diff
@@ -1,20 +1,40 @@
 import { GuidePanel, Heading } from '@navikt/ds-react';
 
 import css from './veilederhilsen.module.css';
+import { SanityDokument } from '~/typer/sanity/sanity';
+import TekstBlokk from '../tekstBlokk/tekstBlokk';
+import { TypografiTyper } from '~/typer/typografi';
 
-export default function VeilederHilsen() {
+interface VeilederHilsenProp {
+  punktlisteDokument: SanityDokument | undefined;
+  tittelPunktlisteDokument: SanityDokument | undefined;
+  spraak: string;
+}
+
+const VeilederHilsen: React.FC<VeilederHilsenProp> = ({
+  punktlisteDokument,
+  tittelPunktlisteDokument,
+  spraak,
+}: VeilederHilsenProp) => {
   return (
     <GuidePanel poster className={`${css.poster}`}>
       <Heading level="2" size="xlarge" className={`${css.tittelMargin}`}>
         Hei [fornavn]
       </Heading>
-      Du må melde fra til oss hvis:
+      <TekstBlokk
+        tekstblokk={tittelPunktlisteDokument}
+        valgBlock={spraak}
```

Comment:
Tenker at språk bør være global. Først og fremst fordi da unngår vi muligheten til at tekster har forskjellig språk, i tillegg slipper vi å sende det rundt overalt.

Reply:
Nå er det bare en variabel for språk, så legger jeg inn en kommentar på issuet for språk at det bør være globalt